### DISCUSSION TITLE
Areas of improvement

### DISCUSSION THREAD

**Discussion by reinterpretcat** [reinterpretcat/vrp#82] [Polls]
Based on your experience of using this project, what would be next good thing to work on?

**Comment by ivannovazzi** [comment]
One gap that comes up when using VRP solvers in practice: once you have an optimal route plan, there's often no easy way to *simulate* or *visualize* the vehicles actually executing those routes in real time — useful for demos, testing dispatch dashboards, or validating downstream systems.

[Moveet](https://github.com/ivannovazzi/moveet) is an open-source fleet simulator that might complement vrp well for this: you load a real road network (OpenStreetMap/GeoJSON), define vehicle waypoints (e.g. from a VRP solution), and it drives synthetic vehicles along those routes using A* pathfinding, streaming live positions over WebSocket.

Not a solver itself, but a useful layer for making computed routes come alive for testing and visualization purposes.
